shared/styles/css.js.flow

```diff
@@ -265,7 +265,7 @@ export type _StylesDesktop = {|
   left?: Dimension,
   letterSpacing?: string,
   lineBreak?: string,
-  lineHeight?: string,
+  lineHeight?: Dimension,
```

Comment:
Guessing we don't want to type line heights as strings?

Reply:
i mentioned this in the other pr i think we don't